demos/dermatology.html

```diff
@@ -253,10 +263,16 @@
             function displayProperties(sourceWidget) {
                 if (document.getElementById("showProperties").checked) {
                     sourceWidget = sourceWidget || currWidget;
-                    propEditor
-                        .data([sourceWidget])
-                        .render()
-                    ;
+                    propEditor.widget(sourceWidget);
+                    propEditor.render();
+                }
+            }
+            
+            function displayPropertyTree(sourceWidget) {
+                if (document.getElementById("showPropertyTree").checked) {
+                    sourceWidget = sourceWidget || currWidget;
+                    propTree.data(propEditor.getDendrogramObject());

```

Comment:
Rename "getDendrogramObject" "getDataTree" (I am trying to convey that you are returning the data structure as a ITree compatible data structure)?